internal/cephfs/volume.go

```diff
@@ -248,8 +254,15 @@ func purgeVolume(ctx context.Context, volID volumeID, cr *util.Credentials, volO
 	if force {
 		arg = append(arg, "--force")
 	}
+	retainSnap, err := validateSubvolumeFeature(ctx, "snapshot-retention", volID, cr, volOptions)
```

Comment:
can we rename `validateSubvolumeFeature` to `IsSubVolumeHasFeature` to better readability?

Reply:
maybe `checkSubvolumeHasFeature` ?

Reply:
sounds good.